packages/flutter/test/cupertino/segmented_control_test.dart

```diff
@@ -1635,4 +1667,94 @@ void main() {
       kIsWeb ? SystemMouseCursors.click : SystemMouseCursors.basic,
     );
   });
+
+  testWidgets('Tap on disabled segment should not change its state', (WidgetTester tester) async {
+    final Map<int, Widget> children = <int, Widget>{
```

Comment:
This seems the same as the `buildOneDisabledSegmentedControl` method, maybe we can reuse it to reduce duplication

Reply:
This is actually not the same as I need `sharedValue` to verify the disabled segment state after being tapped on. Since there is only one place calling `buildOneDisabledSegmentedControl`, I think I will remove it and write the pumped widget as usual.